src/deroff.rs

```diff
@@ -1,8 +1,40 @@
 /// A translation of https://github.com/fish-shell/fish-shell/blob/e7bfd1d71ca54df726a4f1ea14bd6b0957b75752/share/tools/deroff.py
 // """ Deroff.py, ported to Python from the venerable deroff.c """
+use std::collections::HashMap;
+
+type TODO_TYPE = u8;
+type TODO_NUMBER_TYPE = i8;
+
+const OPTIONS: u8 = 0;
```

Comment:
Looks like these three constants are exclusively used by the `tblstate` field (and _it_ only uses these three constants). Maybe we could convert them into
```rust
enum TblState {
    Options,
    Format,
    Data,
}
```
and change the `tblstate` field's type to `TblState`.